src/main/java/cms/gov/madie/measure/resources/TestCaseController.java

```diff
@@ -154,6 +156,23 @@ public ResponseEntity<List<TestCaseImportOutcome>> importTestCasesQdm(
 
     for (TestCaseImportRequest request : testCaseImportRequests) {
       request.setPatientId(UUID.randomUUID());
+      // append given and family name to the import object to report to outcome
+      try {
+        String familyName =
+            testCaseService.getPatientFamilyName(ModelType.QDM_5_6.getValue(), request.getJson());
+            request.setFamilyName(familyName);
+        String givenName =
+            testCaseService.getPatientGivenName(ModelType.QDM_5_6.getValue(), request.getJson());
+            request.setGivenNames(Collections.singletonList(givenName));
+      } catch (JsonProcessingException ex) {
+        log.info(
+            "User {} is unable to import test case with patient id : "
+                + "{} because of JsonProcessingException: "
+                + ex.getMessage(),
+            userName,
+            request.getPatientId());
+        throw new RuntimeException(ex.getMessage());
```

Comment:
codacy is complaining about this exception. you can use `InvalidRequestException`